.github/workflows/update-deploy-single-prod-site.yaml

```diff
@@ -0,0 +1,48 @@
+name: Update and Deploy Site in Another Organization
+
+on:
+  workflow_dispatch: # Allows manual trigger
```

Comment:
Consider making the target version [an input to the workflow](https://github.blog/changelog/2021-11-10-github-actions-input-types-for-manual-workflows/) instead of reading it from `sites.yaml`.